packages/boot/test/bootstrapTests/supports.ts

```diff
@@ -215,13 +215,23 @@ export const makeProposalExtractor = ({ childProcess, fs }: Powers) => {
   const getPkgPath = (pkg, fileName = '') =>
     new URL(`../../../${pkg}/${fileName}`, import.meta.url).pathname;
 
-  const runPackageScript = async (pkg, name, env) => {
-    console.warn(pkg, 'running package script:', name);
-    const pkgPath = getPkgPath(pkg);
-    return childProcess.execFileSync('yarn', ['run', name], {
-      cwd: pkgPath,
+  const importSpec = spec =>
+    importMetaResolve(spec, import.meta.url).then(u => new URL(u).pathname);
+
+  const runPackageScript = async (outputDir, scriptPath, env) => {
+    console.warn('running package script:', scriptPath);
```

Comment:
not sure why this was console.warn before since this is how it's supposed to work.
```suggestion
    console.info('running package script:', scriptPath);
```